crypto/rand/randfile.c

```diff
@@ -208,8 +208,11 @@ int RAND_write_file(const char *file)
          * should be restrictive from the start
          */
         int fd = open(file, O_WRONLY | O_CREAT | O_BINARY, 0600);
-        if (fd != -1)
+        if (fd != -1) {
```

Comment:
As you're touching this, could you please add an empty line before this one?

Reply:
Also style wants on at 186, I think... do you want a second commit or a force-push on my branch? @t8m

Reply:
I just did a force-push since this is such a trivial change.